pom.xml

```diff
@@ -208,6 +208,11 @@
             <artifactId>jedis</artifactId>
             <version>2.9.0</version>
         </dependency>
+        <dependency>
+            <groupId>com.googlecode.xmemcached</groupId>
```

Comment:
Is this a dep for the AWS 'smart" client?

Reply:
right: https://github.com/killme2008/xmemcached/wiki/User-Guide#aws-elasticcache-client